tests/e2e/app-elements/item-details.map.ts

```diff
@@ -5,17 +5,17 @@ const defaultFieldLocator = ".-sf-short-text-default";
 
 export class ItemDetailsMap {
     public static ExtendedTitleField: ElementFinder = element(by.css(".custom-title-input"));
-    public static EditorInternalField: ElementFinder = element(by.cssContainingText("div", "Hello everyone"));
+    public static EditorInternalField: ElementFinder = element.all(by.cssContainingText("div", ITEM_CONTENT_BEGINNING)).first();
     public static HtmlFieldExpandButton: ElementFinder = element(by.css("a[title=Expand]"));
     public static TitleField: ElementFinder = element(by.css(defaultFieldLocator));
     public static TitleInput: ElementFinder = ItemDetailsMap.TitleField.element(by.css("input"));
     public static TitleError: ElementFinder = ItemDetailsMap.TitleField.element(by.css("sf-error"));
     public static HtmlFieldExpander: ElementFinder = element(by.css(".sf-expand-button"));
     public static EditorCustomEditMenu: ElementFinder = element.all(by.css(".sf-notification__content")).last();
-    public static MonacoEditor: ElementFinder = element(by.cssContainingText("div", ITEM_CONTENT_BEGINNING));
+    public static MonacoEditor: ElementFinder = element.all(by.cssContainingText("div", ITEM_CONTENT_BEGINNING)).first();
```

Comment:
Doesn't **element(by.cssContainingText("div", ITEM_CONTENT_BEGINNING));** does the same?

Reply:
It does but logs a warning message in the console that multiple elements were found and the first will be used. This way we get rid of the warning.
It is a fine line between finding the exact elements and not couple the tests with the html structure of Iris components.